pkgs/by-name/mi/miru/update.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl jq gnused
+
+set -euo pipefail
+
+ROOT="$(dirname "$(readlink -f "$0")")"
+if [[ ! "$(basename $ROOT)" == "miru" || ! -f "$ROOT/package.nix" ]]; then
+    echo "error: Not in the miru folder" >&2
+    exit 1
+fi
```

Comment:
nixpkgs-update and r-ryantm will usually call the update script from nixpkgs root folder. You can test this easily with `nix-update -u miru`

Reply:
@pbsds left another comment in #329752 - r-ryantm stopped calling the update script for some reason after than change?

Reply:
please note the `-u`, which will call the `passthru.updateScript`